src/content/concepts/mode.md

```diff
@@ -26,12 +26,13 @@ or pass it as a [CLI](/api/cli/) argument:
 webpack --mode=production
 ```
 
-The following string values are supported:
+This value is expected to always be set, and if not set webpack will warn and fallback to `production` mode.  The following string values are supported:
```

Comment:
I think we should rephrase this, webpack does not expect you to change it if you don't want to, but works for production bundles as default.

Myabe we can express this like: _webpack sets `"production"` as a default for `mode` and warn about to let you know about this behavior. The supported values for `mode` are:_

Reply:
I was unsure about this text too.  Here's the warning message:

```
WARNING in configuration
The 'mode' option has not been set. Set 'mode' option to 'development' or 'production' to enable defaults for this environment.
```

My reading of this is that it implies that the default behavior is `none`, not `production`, since it implies that defaults are not set.  I can update to your copy, but that's currently inaccurate, as it *doesn't* let you know about that behavior.  However, the solve to this is pretty simple -- we should update the warning itself to be clearer.

Reply:
Message says that is not been set, default behavior is `production`, no `none`.
Yes, that message should say that it uses `production` by default